<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1015</width>
    <height>630</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Xen Manager</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="minimumSize">
       <size>
        <width>1015</width>
        <height>569</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Plain</enum>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="sizeConstraint">
         <enum>QLayout::SetDefaultConstraint</enum>
        </property>
        <item>
         <widget class="QLineEdit" name="treeFilter">
          <property name="maximumSize">
           <size>
            <width>320</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="placeholderText">
           <string>Filter</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeView" name="treeView">
          <property name="minimumSize">
           <size>
            <width>128</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>320</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTabWidget" name="tabWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>100</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="tab_3">
        <attribute name="title">
         <string>General</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_4">
        <attribute name="title">
         <string>Storage</string>
        </attribute>
        <layout class="QVBoxLayout" name="verticalLayout"/>
       </widget>
       <widget class="QWidget" name="tab_2">
        <attribute name="title">
         <string>Network</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_5">
        <attribute name="title">
         <string>NICs</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_6">
        <attribute name="title">
         <string>Console</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_7">
        <attribute name="title">
         <string>Performance</string>
        </attribute>
       </widget>
       <widget class="QWidget" name="tab_8">
        <attribute name="title">
         <string>Logs</string>
        </attribute>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1015</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>&amp;File</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_View">
    <property name="title">
     <string>&amp;View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Pool">
    <property name="title">
     <string>&amp;Pool</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Server">
    <property name="title">
     <string>&amp;Server</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_VM">
    <property name="title">
     <string>&amp;VM</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Storage">
    <property name="title">
     <string>&amp;Storage</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Templates">
    <property name="title">
     <string>&amp;Templates</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuT_ools">
    <property name="title">
     <string>T&amp;ools</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Windows">
    <property name="title">
     <string>&amp;Windows</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu_View"/>
   <addaction name="menu_Pool"/>
   <addaction name="menu_Server"/>
   <addaction name="menu_VM"/>
   <addaction name="menu_Storage"/>
   <addaction name="menu_Templates"/>
   <addaction name="menuT_ools"/>
   <addaction name="menu_Windows"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionBack"/>
   <addaction name="actionForward"/>
   <addaction name="separator"/>
   <addaction name="actionAddNewServer"/>
   <addaction name="separator"/>
   <addaction name="actionNewPool"/>
   <addaction name="actionNewStorage"/>
   <addaction name="actionNewVM"/>
   <addaction name="separator"/>
   <addaction name="actionStart"/>
   <addaction name="actionSuspend"/>
   <addaction name="actionReboot"/>
   <addaction name="actionShutdown"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionBack">
   <property name="text">
    <string>Back</string>
   </property>
  </action>
  <action name="actionForward">
   <property name="text">
    <string>Forward</string>
   </property>
  </action>
  <action name="actionAddNewServer">
   <property name="text">
    <string>Add New Server</string>
   </property>
  </action>
  <action name="actionNewPool">
   <property name="text">
    <string>New Pool</string>
   </property>
  </action>
  <action name="actionNewStorage">
   <property name="text">
    <string>New Storage</string>
   </property>
  </action>
  <action name="actionNewVM">
   <property name="text">
    <string>New VM</string>
   </property>
  </action>
  <action name="actionShutdown">
   <property name="text">
    <string>Shutdown</string>
   </property>
  </action>
  <action name="actionReboot">
   <property name="text">
    <string>Reboot</string>
   </property>
  </action>
  <action name="actionSuspend">
   <property name="text">
    <string>Suspend</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="text">
    <string>Start</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
